src/uas/UAS.cc

```diff
@@ -2827,21 +2825,6 @@ void UAS::enableHilXPlane(bool enable)
         }
         qDebug() << "CREATED NEW XPLANE LINK";
         simulation = new QGCXPlaneLink(this);
-
-        float noise_scaler = 0.1f;

```

Comment:
> 1) What's running at 50Hz? I show Xplane9 at 120Hz.
Most people get only 50 Hz

> 2) Does FlightGear talk to QGC at 400Hz? (and how do I reproduce your 400Hz HIL test?)
JSBSim via UDP could, if you were to implement it. Also Eddy seems to be able to run much faster.

> 3) Where does the sensor noise get injected when QGC is bypassed?
It doesn't. It needs a central simulation hub, which is why I wanted to break it out from QGC.

> 4) How do I run HIL without QGC?
Read the JSBSim HIL docs I linked above.